spec/features/deck_spec.rb

```diff
@@ -17,7 +17,7 @@
       click_link 'Все колоды'
       click_link 'Добавить колоду'
       fill_in :deck_name, with: 'Тест'
-      click_button 'Создать'
+      click_button 'Создать колоду'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.